lms/djangoapps/instructor/views/api.py

```diff
@@ -2327,6 +2327,32 @@ def list_financial_report_downloads(_request, course_id):
     return JsonResponse(response_payload)
 
 
+# Is the non_atomic_requests header correct?
```

Comment:
I think this header is correct, it seems to match the other methods in this api at least.

It makes sense to me that a task should be able to update it's data model during execution of this request, so I can see why it's here.